index.html

```diff
@@ -0,0 +1,532 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Njoroge</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <section id="toolbar">
+        <div id="toolbar-container"></div>
+        
+        <div id="toolbar-logo-placeholder">
+            <p id="toolbar-logo-name">Njoroge</p>
+        </div>
+
+        <div id="toolbar-menu-button">
+            <div id="toolbar-menu-icon">
+                <div id="toolbar-menu-union-icon">
+                </div>
+            </div>
+        </div>
+        
+    </section>
+
+    <section id="headline">
+        <h3 class="my-name">Hey There.<br>I'm Njoroge</h3>
+        
+        <h4 class="supporting-text">I'm a Software Developer</h4>
+
+        <p class="description">I can help you build a product , feature or<br>
+            website Look through some of my work<br>
+            and experience! If you like what you see<br>
+            and have a project you need developed,<br> please 
+            do not hestiate to contact me.</p>
+
+        <div class="social-media">
+            <!-- github -->
+            <div class="github frame">
+                <div class="github-icon">
+                    <a href="https://github.com/francis450"><img src="img/social-media-icons/github.svg" alt=""></a>
+                </div>
+            </div>
+
+            <!-- linkedin -->
+            <div class="linkedin frame">
+                <div class="linkedin-icon">
+                    <a href="https://linkedin.com/in/njoroge-francis"><img src="img/social-media-icons/linkedin.svg" alt=""></a>
+                </div>
+            </div>
+
+            <!-- peace -->
+            <div class="peace frame">
+                <div class="peace-icon">
+                    <a href="#"><img src="img/social-media-icons/peace.svg" alt=""></a>
+                </div>
+            </div>
+
+            <!-- twitter -->
+            <div class="twitter frame">
+                <div class="twitter-icon">
+                    <a href="https://twitter.com/nkamandefrancis"><img src="img/social-media-icons/twitter.svg" alt=""></a>
+                </div>
+            </div>
+
+            <!-- m -->
+            <div class="m frame">
+                <div class="m-icon">
+                    <a href="https://medium.com/@franciskamande2001"><img src="img/social-media-icons/m.svg" alt=""></a>
+                </div>
+            </div>
+        </div>
+
+        <div class="background-image">
+            <img src="img/headline-background.svg" alt="">
+        </div>
+    </section>
+
+    <!-- Works -->
+    <section id="recent-works">
```

Comment:
- [ ] Good job layering your work section with flex 👍🏽 . According to the project requirement, kindly ensure that the main container of the projects has a` display:grid` instead of flex. The goal is to get you accustomed to how to use CSS grid.